package/scripts/params.py

```diff
@@ -119,6 +119,15 @@
 
 router_hosts = config['clusterHostInfo']['cdap_router_hosts']
 router_hosts.sort()
-cdap_router_host = router_hosts[0]
+if len(router_hosts) > 1:
+    for i, val in enumerate(router_hosts):
```

Comment:
seems like the ``enumerate`` is not needed... ``i`` is not used.  i think you can just do: ``for val in router_hosts:``